crime-assessment-service/src/main/java/uk/gov/justice/laa/crime/assessmentservice/iojappeal/validator/ApiCreateIojAppealRequestValidator.java

```diff
@@ -25,18 +25,19 @@ public class ApiCreateIojAppealRequestValidator {
 
     public static final String METADATA = "Metadata";
     public static final String CASE_MANAGEMENT_UNIT = "Case Management Unit";
-    public static final String APP_IDS = "Both Application Id and Legacy Application Id";
+    public static final String LEGACY_APPLICATION_ID = "Legacy Application Id";
     public static final String USER_SESSION = "User Session";
     public static final String SESSION_ID = "Session ID";
     public static final String USERNAME = "Username";
 
     public static final String APPEAL = "Appeal";
-    public static final String APPEAL_DECISION = "Appeal Decision";
+    public static final String APPEAL_SUCCESSFUL = "Appeal Successful";
     public static final String APPEAL_ASSESSOR = "Appeal Assessor";
     public static final String APPEAL_REASON = "Appeal Reason";
     public static final String DECISION_REASON = "Decision Reason";
     public static final String DECISION_DATE = "Decision Date";
     public static final String RECEIVED_DATE = "Received Date";
+    public static final String APPLICATION_RECEIVED_DATE = "Application Received Date";
     public static final String APPEAL_REASON_HARDIOJ = "HARDIOJ";
```

Comment:
Hopefully we can get rid of all of this stuff if we're calling the v2 endpoints that use the new schemas.